tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -2095,3 +2095,263 @@ def call(self, inputs, state):
 
     new_state = rnn_cell_impl.LSTMStateTuple(c, m)
     return m, new_state
+
+
+class MultiplicativeIntegrationRNNCell(rnn_cell_impl.RNNCell):
```

Comment:
Can you make this subclass LayerRNNCell, and move all the get_variable() calls into a "def build(input_shape):".  at the end of the build function set `self.built = True`.  In `call`, only access variables via `self` (i.e., `self._kernel`, `self._bias`, etc).  See the current LSTMCell and GRUCell implementations to see what I mean.